Dockerfile

```diff
@@ -196,6 +197,9 @@ COPY ./target/bin /usr/local/bin
 COPY ./target/check-for-changes.sh ./target/start-mailserver.sh ./target/fail2ban-wrapper.sh ./target/postfix-wrapper.sh ./target/docker-configomat/configomat.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/*
 
+# Setup AWS SES Fetch Mail
+RUN (echo "*/1 * * * * /usr/local/bin/fetch-aws-ses-mail"; crontab -l) | crontab -
```

Comment:
Should I do this while booting in the mailserver-start-script?

Reply:
Yes you should.